packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -116,9 +133,10 @@ async function fetchDestinations(
 
   return callDestinationEndpoint(targetUri, headers, options)
     .then(response => {
-      const destinations: Destination[] = response.data.map(d =>
-        parseDestination(d)
-      );
+      const destinations: Destination[] = response.data
+        .filter(isParsable)
```

Comment:
looks much cleaner 👍 

I was not aware of this short-syntax for filter where you don't need to write the parameter 💯

Reply:
function handler passing like in C++ :-)

Reply:
[le](https://xkcd.com/138/)